app/static/style.css

```diff
@@ -96,6 +96,16 @@ p {
   border: none;
   background-color: whitesmoke;
 }
+  
+.error-message {
+    line-height: 0;
+    color: red;
+    padding-left: 12.5rem;
+}
+
+.subtitle {
+    font-size: 20px;
```

Comment:
Use rem

Reply:
that's not my code.. must be from merging..